backend/src/main/resources/db/migration/V1__create_first_scheme.sql

```diff
@@ -102,3 +103,27 @@ CREATE TRIGGER tgr_answers_set_updated
     ON answers
     FOR EACH ROW
 EXECUTE PROCEDURE set_updated();
+
+-- Create table users
+CREATE TABLE users
+(
+    id            VARCHAR(36) UNIQUE  NOT NULL,
+    created       timestamptz         NOT NULL,
+    updated       timestamptz,
```

Comment:
Pridej sem teda do vsech updated NOT NULL, at je to konzistentni s Kotlin kodem.